diff --git trunk/jdmarker.c-orig trunk/jdmarker.c
index 6fc0f7d..e65c413 100644
--- trunk/jdmarker.c-orig
+++ trunk/jdmarker.c
@@ -911,7 +911,7 @@ next_marker (j_decompress_ptr cinfo)
   }
 
   if (cinfo->marker->discarded_bytes != 0) {
-    WARNMS2(cinfo, JWRN_EXTRANEOUS_DATA, cinfo->marker->discarded_bytes, c);
+    /* WARNMS2(cinfo, JWRN_EXTRANEOUS_DATA, cinfo->marker->discarded_bytes, c); */
     cinfo->marker->discarded_bytes = 0;
   }
 
